packages/cli/generators/repository/templates/src/repositories/repository-crud-default-template.ts.ejs

```diff
@@ -0,0 +1,14 @@
+import {<%= repositoryTypeClass %>, juggler} from '@loopback/repository';
+import {<%= modelName %>} from '../models';
+import {inject} from '@loopback/core';
+
+export class <%= className %>Repository extends <%= repositoryTypeClass %><
+  <%= modelName %>,
+  typeof <%= modelName %>.prototype.<%= idType %>
+> {
+  constructor(
+    @inject('datasources.<%= dataSourceName %>') protected datasource: juggler.DataSource,
```

Comment:
Can we make this consistent with `repository-kv-template.ts.ejs` please and use `dataSourceImportName` instead of `juggler.DataSource`?

Also `dataSourceImportName` does not sound as a correct description to me, how about `dataSourceClassName`?

Reply:
dataSourceClassName sounds perfect.